### PR TITLE
<Icons/> - small size icons

### PR DESCRIPTION
**Pull request by mosheKerbel**
According to issue https://github.com/wix-private/wsr-issues/issues/358
New icons were added to wix-ui-icons-common - https://github.com/wix/wix-ui/pull/1252

This PR modify components storybooks which may include an icon. The addition is to describe and give an example for the user for the different size icons usage.

### CODE REVIEW

**Comment by mosheKerbel** [comment]
@shlomitc @argshook

File: src/Badge/docs/index.story.js [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+  header,
+  importExample,
+  playground,
+  tab,
+  tabs,
+  testkit,
+} from 'wix-storybook-utils/dist/src/Sections';
+import allComponents from '../../../stories/utils/allComponents';
+
+import LiveCodeExample from '../../../stories/utils/LiveCodeExample';
```
Comment (argshook) [inline src/Badge/docs/index.story.js]: please use `code` section

File: stories/TextInput/examples.js [hunk 2]
```diff
@@ -39,6 +39,24 @@ export const affix = `
 </Layout>
 `;
 
+export const iconAffix = `
+<Layout>
+  <C      <Input size="small" placeholder="Placeholder" prefix={<Input.ChevronRightSmall><MoreSmall/></Input.IconAffix>}/>
```
Comment (argshook) [inline stories/TextInput/examples.js]: copy paste typo

File: src/Badge/docs/index.story.js (hunk 1 above)
Comment (mosheKerbel) [reply src/Badge/docs/index.story.js]: done

**Pull request merged**